frontend/src/metabase/css/dashboard.css

```diff
@@ -226,8 +213,9 @@
   height: 40px;
   bottom: 0;
   right: 0;
-  cursor: nwse-resize;
+  cursor: nwse-resize !important;
```

Comment:
Yes, I am afraid these `!important` will come back to haunt us.

Is it looking very costly to write things without it?

Reply:
I guess so, as we need to overwrite [the style coming for `react-grid-layout`](https://github.com/react-grid-layout/react-grid-layout/blob/5cf8afb29dbf2ab373378e775219488d4e63454e/css/styles.css#L67)

Reply:
Gotcha, thanks.

Reply:
Maybe a comment about that would clarify for future readers.

Reply:
Good point! Will add, thank you

Reply:
It's a pity they (RGL) use hard coded CSS values instead of either css vars or scss. It would've been much easier to configure and tailor it to the custom needs. And we could've avoided using `!important`.